test/acceptance/keywords/tables/table_should_not_contain.robot

```diff
@@ -0,0 +1,120 @@
+    ${col}=    set variable    1
+    ${value} =    set variable    simpleTable_A1
+    ${err}=    Set Variable    Cell in table '${table}' in row #${row} and column #${col} should not have contained text '${value}'.
+    Run Keyword And Expect Error    ${err}    table cell should not contain    ${table}    ${row}    ${col}    ${value}
+
+Test Table Row Should Not Contain
+    [Documentation]    Test Table Row Should Not Contain
+    ${table}=    set variable    simpleTable
+    ${row}=    set variable    1
+    ${value} =    set variable    simpleTable_A2
+    table row should not contain    ${table}    ${row}      ${value}
+
+Test Table Row Should Give Error When It Contains
+    [Documentation]    Test Table Row Should Give Error When It Contains
+    ${table}=    set variable    tableWithSingleHeader
+    ${value} =    set variable    tableWithSingleHeader_A1
+    ${row}=    set variable    1
+    ${err}=    Set Variable    Row #${row} in table identified by '${table}' should have contained text '${value}'.
+    run keyword and expect error  ${err}    table row should not contain  ${table}    ${row}    ${value}
+
+Test Table column Should Not Contain
+    [Documentation]    Test Table column Should Not Contain
+    ${table}=    set variable    simpleTable
+    ${column}=    set variable    1
+    ${value} =    set variable    simpleTable_B2
+    table column should not contain    ${table}    ${column}      ${value}
+
+Test Table column Should Give Error When It Contains
+    [Documentation]    Test Table column Should Give Error When It Contains
+    ${table}=    set variable    tableWithSingleHeader
+    ${column}=    set variable    1
+    ${value} =    set variable    tableWithSingleHeader_A1
+    ${err}=    Set Variable    Column #${column} in table identified by '${table}' should not have contained text '${value}'.
+    run keyword and expect error  ${err}    table column should not contain  ${table}    ${column}      ${value}
+
+Test Table Header Should Not Contain
+    [Documentation]    Test Table Header Should Not Contain
+    ${table}=    set variable    tableWithSingleHeader
+    ${value} =    set variable    tableWithSingleHeader_A3
+    table header should not contain  ${table}    ${value}
+
+Test Table Header Should Give Error When It Contains
+    [Documentation]    Test Table Header Should Give Error When It Contains
+    ${table}=    set variable    tableWithSingleHeader
+    ${value} =    set variable    tableWithSingleHeader_A1
+    ${err}=    Set Variable    Header in table identified by '${table}' should not have contained text '${value}'.
+    run keyword and expect error  ${err}    table header should not contain  ${table}    ${value}
+
+Test Table Footer Should Not Contain
+    [Documentation]    Test Table Footer Should Not Contain
+    ${table}=    set variable    withHeadAndFoot
+    ${expected} =    set variable    withHeadAndFoot_A1
+    table footer should not contain    ${table}    ${expected}
```

Comment:
The value *withHeadAndFoot_A1* is already in the body, should i just add one more verification where value is not in header and rename the test? or should I rename this test to verify that value is in body but not in footer and then add another test to verify where value is in footer but not in header?